webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala

```diff
@@ -735,6 +740,242 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
+                 Right("ok")
+               } else {
+                 Left(s"Please check that file '${hook.pathAsString}' exists and is readable and writtable.")
+               }
+       _ <- props.maxNodes
+       _ <- props.maxNodes
+      } yield {
+        ()
+      }).swap.toOption
+    }
+
+    (xml : NodeSeq) => {
+      val hook = File(hookPath)
+      var hookProps = readProp()
+
+      // initial errors
+      (hookProps.maxNodes :: hookProps.percent :: Nil).map(_.swap.toOption).flatten.distinct match {
+        case Nil => //nothing
+        case seq => S.notice("generationHookTriggerNodeUpdateMsg", seq.mkString("; "))
+      }
+
+      // if the hook file is missing, display an error message
+      hasError(hook, hookProps) match {
+        case Some(error) =>
+          ("#generationHookTriggerNodeUpdateForm" #> ("There was an error when trying to access the file. It can not be configured. " +
+                                                    s"Error was: ${error}"
+                                                    )
+          ) apply(xml)
+        case None =>
+          var isEnabled = hook.isExecutable
+          var currentIsEnabled = isEnabled
+          var currentMax = hookProps.maxNodes.getOrElse(1000)
+          var currentMaxStr = currentMax.toString
+          var currentPercent = hookProps.percent.getOrElse(1000)
+          var currentPercentStr = currentPercent.toString
+
+          def noModif() = {
+            isEnabled == currentIsEnabled && TriggerProp(Right(currentMax), Right(currentPercent)) == hookProps
+          }
+          def check() = {
+            val msg = scala.collection.mutable.Buffer[String]()
+            try {
+              currentMax = currentMaxStr.toInt
+              if(currentMax < 0) {
+                 msg += "Error: max number of nodes must be positive"
+              }
+            } catch {
+              case ex: NumberFormatException => msg += "Error: max number of node must be an int"
+            }
+            try {
+              currentPercent = currentPercentStr.toInt
+              if(currentPercent < 0 || currentPercent > 100) {
+                 msg += "Error: percent of nodes must be in interval [0,100]"
+              }
+            } catch {
+              case ex: NumberFormatException => msg += "Error: percent of nodes must be an int"
+            }
+            S.notice("generationHookTriggerNodeUpdateMsg", <span class="error">{msg.mkString("; ")}</span>)
+            Run(s"""$$("#generationHookTriggerNodeUpdateSubmit").attr("disabled",${noModif});""")
+          }
+
+          def submit() = {
+            S.notice("generationHookTriggerNodeUpdateMsg", saveAll(hook, currentIsEnabled, currentMax, currentPercent) match {
+              case Right(())  =>
+                isEnabled = currentIsEnabled
+                hookProps = TriggerProp(Right(currentMax), Right(currentPercent))
+                "'check generated policies' property updated"
```

Comment:
Should not be 'check generated policies' in the whole section but "trigger node update"